tests/basic/kubevirt_test.go

```diff
@@ -0,0 +1,123 @@
+package tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	"github.com/portworx/sched-ops/task"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/log"
+	"time"
+
+	. "github.com/portworx/torpedo/tests"
+)
+
+var _ = Describe("{AddNewDiskToKubevirtVM}", func() {
```

Comment:
@spai-px - looks like we will be using these methods a lot as I can see few test cases based on this. Can we have this test case as a common function itself which basically, counts current disks, attaches "n" number of new disks and then recounts the total disks again.

Reply:
Yeah this seems like we can move it to a function. I have created a function AddDisksToKubevirtVM and created a new file in tests folder named as kubevirt_helper.go.

Reply:
As discussed have moved the implementation to drivers folder. under kubevirt folder